pkg/ipamd/datastore/data_store.go

```diff
@@ -35,9 +36,9 @@ const (
 	// minENILifeTime is the shortest time before we consider deleting a newly created ENI
 	minENILifeTime = 1 * time.Minute
 
-	// addressCoolingPeriod is used to ensure an IP not get assigned to a Pod if this IP is used by a different Pod
-	// in addressCoolingPeriod
-	addressCoolingPeriod = 30 * time.Second
+	// envIPCooldownPeriod is used to ensure an IP not get assigned to a Pod if this IP is used by a different Pod
+	// in envIPCooldownPeriod
```

Comment:
nit: let's clean up this comment to say: "envIPCooldownPeriod (default 30 seconds) specifies the time after pod deletion before an IP can be assigned to a new pod"